README.md

```diff
@@ -1,220 +1,130 @@
-        testnet=1
-        whitelist=0.0.0.0/0
-        server=1
-        rpcuser=rpcuser
-        rpcpassword=rpcpass
-    expose:
-      - "18332"
-    ports:
-      - "0.0.0.0:18333:18333"
-    volumes:
-      - "bitcoin_datadir:/data"
-
-  clightning_bitcoin:
-    image: elementsproject/lightningd
-    container_name: lightningd
-    command:
-      - --bitcoin-rpcconnect=bitcoind
-      - --bitcoin-rpcuser=rpcuser
-      - --bitcoin-rpcpassword=rpcpass
-      - --network=testnet
-      - --alias=myawesomenode
-      - --log-level=debug
-    environment:
-      EXPOSE_TCP: "true"
-    expose:
-      - "9735"
-    ports:
-      - "0.0.0.0:9735:9735"
-    volumes:
-      - "clightning_bitcoin_datadir:/root/.lightning"
-      - "bitcoin_datadir:/etc/bitcoin"
-    links:
-      - bitcoind
-
-volumes:
-  bitcoin_datadir:
-  clightning_bitcoin_datadir:
+```bash
+sudo apt-get install -y software-properties-common
+sudo add-apt-repository -u ppa:bitcoin/bitcoin
+sudo add-apt-repository -u ppa:lightningnetwork/ppa
+sudo apt-get install bitcoind lightningd
 ```
 
 ### Starting `lightningd`
 
-In order to start `lightningd` you will need to have a local `bitcoind`
-node running in either testnet or regtest mode:
+In order to start `lightningd` you will need to have a local `bitcoind` node running (in this case we start `testnet`):
 
-    bitcoind -daemon -testnet
+```bash
+bitcoind -daemon -testnet
+```
 
 Wait until `bitcoind` has synchronized with the testnet network.
 
-Make sure that you do not have `walletbroadcast=0` in your
-`~/.bitcoin/bitcoin.conf`, or you may run into trouble.
-Notice that currently pruned nodes are not supported and may result in
-`lightningd` being unable to synchronize with the blockchain.
+Make sure that you do not have `walletbroadcast=0` in your `~/.bitcoin/bitcoin.conf`, or you may run into trouble.
+Notice that running `lightningd` against a pruned node may cause some issues if not managed carefully, see [below](#pruning) for more information.
 
 You can start `lightningd` with the following command:
 
-    lightningd/lightningd --network=testnet --log-level=debug
+```bash
+lightningd --network=testnet --log-level=debug
+```
+
+Please refer to `lightningd --help` for all other command line options.
+
+### JSON-RPC Interface
 
-### Listing all commands:
-`cli/lightning-cli help` will print a table of the API and lists the
-following commands
+c-lightning exposes a [JSON-RPC 2.0][jsonrpcspec] interface over a Unix Domain socket located in its home directory (default: `$HOME/.lightning`).
+The Unix Domain Socket has the advantage of not being exposed over the network by default, allowing users to add their own authentication and authorization mechanism, while still providing a fully functional RPC interface out of the box.
+
+You can use `lightning-cli help` to print a table of the available RPC methods that can be called.
+The JSON-RPC interface is also documented in the following manual pages:
```

Comment:
While I like the fact that a couple of manpages are linked I think it is more interesting to tell the people that they can access the manpage of a command with `lightning-cli help COMMAND_NAME` and that `lightning-cli help` only has a short overview for the commands in comparison to the real manpage